app/assets/javascripts/plate-map/bottom-table.js

```diff
@@ -127,7 +127,7 @@ var plateLayOutWidget = plateLayOutWidget || {};
       exportData: function(format) {
         var data = [];
         var rows = document.querySelectorAll("table tr");
-        var corToLocMap = $("#my-plate-layout").plateLayOut("createObject").colorToLoc;
+        var corToLocMap = $("#my-plate-layout").plateLayOut("getCurrentPlate").colorToLoc;
```

Comment:
Same problem here, use `this.getCurrentPlate().colorToLoc;`

Reply:
Agreed and changed